# This message contains event rate information

std_msgs/Header header     # standard ROS header
float64 off_rate  # number of OFF events per second
float64 on_rate   # number of ON  events per second
float64 msg_rate  # number of messages per second
float64 time_bin  # size of time bin in seconds
